src/cli.js

```diff
@@ -38,6 +40,7 @@ export default class Main {
   async run () {
     const updater = new Updater(out)
     const plugins = new Plugins(out)
+    await MigrateV5Plugins.run([], this.config)
```

Comment:
I modeled the migrator more like a `Command` than the `Updater` class b/c of wanting easy access to plugins/configs, but open to changing that.